Tools/eslint-config-cesium/node.js

```diff
@@ -0,0 +1,13 @@
+/*eslint-env node*/
+'use strict';
+
+module.exports = {
+    "extends": "cesium",
```

Comment:
Since this is JS code now and not JSON, we always use single quotes and none of the property names need to be quoted. (Unless this needs to be a certain way for eslint to understand it, which I doubt).